src/Content/Item.php

```diff
@@ -333,7 +339,8 @@ public function localize(array &$item)
 					default:
 						if ($obj['resource-id']) {
 							$post_type = $this->l10n->t('photo');
-							$m=[]; preg_match("/\[url=([^]]*)\]/", $obj['body'], $m);
+							$m = [];
+							preg_match("/\[url=([^]]*)\]/", $obj['body'], $m);
 							$rr['plink'] = $m[1];
```

Comment:
Initialization is superfluous
```suggestion
							preg_match("/\[url=([^]]*)\]/", $obj['body'], $matches);
							$rr['plink'] = $matches[1];
```

Reply:
Done